docs/standardisation-scope.adoc

```diff
@@ -0,0 +1,199 @@
+= Standardisation scope of Cypher
+
+One of the challenges faced by this effort to standardise Cypher has been to choose the set of language features to standardise.
+Cypher is not a new language, but has several years of history, which includes some historical artifacts left in the feature set.
+This is generally not because there hasn't been good replacements, but because Cypher's role has been to support a specific product (Neo4j).
+The lifecycle of a standardised language is different, however, and removal of standardised features is generally hard.
```

Comment:
I'm not so keen on the `which means` here... I prefer the `and`. Is there another alternative?